### PR TITLE
[PRT] Openupgrade framework

### PR DESCRIPTION
**Pull request by StefanRijnhart**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/openupgrade_records/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+# Copyright 2011-2015 Therp BV <https://therp.nl>
+# Copyright 2016 Opener B.V. <https://opener.am>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'OpenUpgrade Records',
+    'version': '11.0.1.0.0',
```
Comment (MiquelRForgeFlow) [inline odoo/addons/openupgrade_records/__manifest__.py]: 12

Comment (StefanRijnhart) [reply odoo/addons/openupgrade_records/__manifest__.py]: Fixed, thanks!

**Comment by StefanRijnhart** [comment]
Analysis that this port creates: https://github.com/OCA/OpenUpgrade/commit/1273884c85f841d6289b1d6941f800e84badc9af

**Comment by StefanRijnhart** [comment]
I synced openupgrade/12.0 with saas-11.5 so I don't know why all these upstream commits still show up here.

**Comment by seb-odoo** [comment]
Hello, why is my name mentioned in this PR?

**Comment by switch87** [comment]
Same here, I have nothing to do with this pr as far as I am aware of.

**Comment by StefanRijnhart** [comment]
@seb-odoo @switch87 sorry about that. You have recent commits merged in odoo/saas-11.5, a copy of which is the target branch of this PR. After first creating this PR, saas-11.5 included changes that I needed here as well so I first updated the target branch with the latest changes to saas-11.5 (including your commits) and then rebased the feature branch. Now for some reason, it took Github hours to figure out that your commits were not actually part of this PR (their changes showed up in this PR's diff until yesterday evening). The OCA CLA-bot then started to check if you have signed the OCA CLA. As a result you are subscribed to this PR. Feel free to unsubscribe though!

**Comment by pedrobaeza** [comment]
Merging this for working on the base module migration.

**Comment by mpdevilleres** [comment]
I notice in this PR that the 12.0 argument is missing, is it intentional?

```
migrations = {
    '11.0': {
        'addons': {
            'addons': {
                'type': 'link',
                'url': os.path.join('server', 'addons'),
            },
        },
        'server': {
            'type': 'git',
            'url': 'git://github.com/OpenUpgrade/OpenUpgrade.git',
            'branch': '11.0',
            'addons_dir': os.path.join('odoo', 'addons'),
            'root_dir': os.path.join(''),
            'cmd': 'odoo-bin --update=all --database=%(db)s '
                   '--config=%(config)s --stop-after-init --no-xmlrpc',
            },
    },
    '10.0': {
        'addons': {
            'addons': {
                'type': 'link',
                'url': os.path.join('server', 'addons'),
            },
        },
        'server': {
            'type': 'git',
            'url': 'git://github.com/OpenUpgrade/OpenUpgrade.git',
            'branch': '10.0',
            'addons_dir': os.path.join('odoo', 'addons'),
            'root_dir': os.path.join(''),
            'cmd': 'odoo-bin --update=all --database=%(db)s '
                   '--config=%(config)s --stop-after-init --no-xmlrpc',
            },
    },
```

**Comment by pedrobaeza** [comment]
That script is not maintained anymore. You should follow manual migration steps: https://doc.therp.nl/openupgrade/migration_details.html

**Comment by mpdevilleres** [comment]
@pedrobaeza thnx for the reference

I got lost in the codes :)

**Pull request merged**